libfreerdp/core/settings.c

```diff
@@ -252,13 +252,27 @@ void settings_load_hkey_local_machine(rdpSettings* settings)
 
 BOOL settings_get_computer_name(rdpSettings* settings)
 {
-	DWORD nSize = MAX_COMPUTERNAME_LENGTH + 1;
-	CHAR computerName[MAX_COMPUTERNAME_LENGTH + 1];
+	DWORD nSize = 0;
+	CHAR* computerName;
+
+	if (GetComputerNameExA(ComputerNameNetBIOS, NULL, &nSize))
```

Comment:
@nfedera Well, the function will return ```ERROR_BUFFER_OVERFLOW``` as ```GetComputerNameA``` is used internally. Don't know if this is a bug or desired?